hail/python/hail/expr/expressions/base_expression.py

```diff
@@ -754,6 +754,116 @@ def _to_relational_preserving_rows_and_cols(self):
             return x.order_by(*x.key).select(name)
         return x
 
+    def export(self, filename, delimiter='\t', header=True):
+        """Export a field to a text file
+
```

Comment:
add the include experimental?

Reply:
I feel like we cop out on long term support with this experimental tag but users don't really pay much attention to it. I'm OK committing to supporting this interface until 0.3.

Reply:
OK, you need to test it then

Reply:
maybe we should add an experimental decorator that warns?